catalog/core/catalog-core-directorymonitor/src/main/java/org/codice/ddf/catalog/content/monitor/DurableFileAlterationListener.java

```diff
@@ -67,16 +71,16 @@ private void init() {
     }
   }
 
-  @Override
-  public void onFileChange(File file) {
-    filesWatcher.watch(new FileWatcher(file, this::fileUpdate));
+  public void onFileChange(File file, Synchronization cb) {
```

Comment:
Why did we remove the `@Override` on the interface methods

Reply:
Do they need to be there? When I switched interfaces I got rid of them for no other reason besides the compiler didn't complain.

Reply:
They are not required but it is good practice